src/Symfony/Component/Form/Util/VirtualFormAwareIterator.php

```diff
@@ -22,11 +20,26 @@
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
- * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use
- *             {@link InheritDataAwareIterator} instead.
+ * @deprecated since version 2.3, to be removed in 3.0.
+ *             Use {@link InheritDataAwareIterator} instead.
  */
 class VirtualFormAwareIterator extends \IteratorIterator implements \RecursiveIterator
 {
+    public function __construct(\Traversable $iterator, $triggerDeprecationNotice = true)
+    {
+        /**
```

Comment:
this should be  regular command, not a docComment IMO, so it should start with ``/*`` only